src/plugins/sqlalchemy_create_object_operator.py

```diff
@@ -104,7 +136,7 @@ def execute(self, context: Dict[str, Any]) -> None:
         else:
             self.data_table_name = self.data_table_name
 
-        data_schema_url = SCHEMA_URL / self.data_schema_name / self.data_schema_name
+        data_schema_url = self.get_schema(self.data_schema_version)
         data = schema_fetch_url_file(data_schema_url)
```

Comment:
In een andere operator heb ik `schema_fetch_url_file` net vervangen. Die gaat nu namelijk stuk op het moment dat het opgevraagde schema relaties heeft. Ook de  dataset van het doel van de relatie moet namelijk geladen worden. Dat doet `schema_def_from_url` wel agv. de nieuwe `prefetch_related` die ik daarin heb toegevoegd.